extensions/calDotCom/actions/getBooking.ts

```diff
@@ -86,6 +90,7 @@ export const getBooking: Action<typeof fields, typeof settings> = {
           status: booking.status,
           cancelUrl: `https://app.cal.com/booking/${booking.uid}?cancel=true`,
           rescheduleUrl: `https://app.cal.com/reschedule/${booking.uid}`,
+          videoCallUrl: booking.metadata.videoCallUrl ?? '',
```

Comment:
Was thinking about whether this should be an empty string or left as undefined, but this is a data point that can actually be null in a 200 response (compared to the other data points which will have a value) -- so this is okay. If the user wants to use this value they can check if it is empty and/or use a fallback value in the dynamic variable instead